js/src/lib/interfaces/Libraries.ts

```diff
@@ -233,14 +233,22 @@ model = timm.create_model("hf_hub:${model.id}", pretrained=True)`;
 const sklearn = (model: ModelData) => {
 	if (model.tags?.includes("skops")) {
 		const skopsmodelFile = model.config?.sklearn?.filename;
+		const skopssaveFormat = model.config?.sklearn?.save_format;
+		if (skopssaveFormat == "pickle")
+		{
+			return `download("${model.id}", "path_to_folder")
```

Comment:
Please add the imports at the top, and also with `from skops.hub_utils import download` which is not in this one